### ISSUE TITLE
Query not working

### ISSUE THREAD

**Issue by callzhang** [jamalex/notion-py#215]
```
filter_params = {
        "filters": [{
            "filter": {
                "operator": "is_empty"
            },
            "property": "title"
        }],
        "operator": "and"
    }
    result = cv.build_query(filter=filter_params).execute()
```
Got full result (expecting none). And other queries result the same.

**Comment by GabrielMMelo** [comment]
I was facing the same here. Seems like it's a package release issue. The last version published on PyPi is `0.0.25`. I've installed manually the current `master` version (`0.0.26`) and it's working like a charm.

**Comment by jamalex** [comment]
Sorry for the delay, will be releasing the new version to PyPI tonight.
